README.md

```diff
@@ -53,6 +53,12 @@ It should be clear that FLoC will never be able to prevent all misuse. There wil
 ### Opting Out of Computation
 A site should be able to declare that it does not want to be included in the user's list of sites for cohort calculation. This can be accomplished via a new `interest-cohort` [permissions policy](https://www.w3.org/TR/permissions-policy-1/). This policy will be default allow. Any frame that is not allowed `interest-cohort` permission will have a default value returned when they call `document.interestCohort()`. If the main frame does not have `interest-cohort` permission then the page visit will not be included in interest cohort calculation.
 
+For example, a site can opt out of all FLoC cohort calculation by sending the HTTP response header:
+
+```
+Permissions-Policy: interest-cohort=()
```

Comment:
@xyaoinum PTAL.  I think this line should be
```
Feature-Policy: interest-cohort 'none'
```
but you're the one who wrote the code...

Reply:
I believe both syntaxes are correct. `Feature-Policy: interest-cohort 'none'` is the legacy way.

Reply:
Ah great.  Want to add a Permissions-Policy test, next to the [Feature-Policy one](https://source.chromium.org/chromium/chromium/src/+/master:chrome/test/data/federated_learning/feature_policy_interest_cohort_none.html.mock-http-headers;l=2?q=interest-cohort&ss=chromium%2Fchromium%2Fsrc&start=11)?  Then merge this PR once you have a test that verifies it works :-).

Reply:
Verified in https://chromium-review.googlesource.com/c/chromium/src/+/2727761